### PR TITLE
new comment change

### PR DESCRIPTION
**Pull request by ZachLalwani**
finished it all

### CODE REVIEW

**Comment by ZachLalwani** [comment]
good job zach!

File: Exercise_A/index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
     />
   </head>
   <body>
-    <!-- CODE HERE -->
+    <a href="about.html"><img src="./assets/flowerShop.png" alt="flowerShop"></a>
 
     <script src="" async defer></script>
```
Comment (samoshaughnessy) [inline Exercise_A/index.html:20]: Well done Zach this  exercise was completed very well

File: Exercise_B/index.css [hunk 2]
```diff
@@ -2,3 +2,27 @@
...
+    margin-left: 45%;
+}
+
+.greenBox {
+    height: 100px;
+    width: 100px;
+    background-color: green;
+    float: right;
+    display: inline-block;
+}
```
Comment (samoshaughnessy) [inline Exercise_B/index.css:28]: Decent implementation for this exercise, though you can achieve this without using margins.
On your red class only have the background colour and:
clear: both;
margin: auto;

But I would extract the width and height into a separate class such that you do not repeat the properties over and over again for each block.

Consider using a box class and id's for blue, red and green

File: Exercise_B/index.html [hunk 3]
```diff
@@ -1,22 +1,34 @@
...
-    <script src="" async defer></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title></title>
+  <meta name="description" content="" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (samoshaughnessy) [inline Exercise_B/index.html:9]: Try to put your custom CSS after bootstrap and other libraries that are being applied to your website.

File: Exercise_C/index.css [hunk 4]
```diff
@@ -2,3 +2,30 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
```
Comment (samoshaughnessy) [inline Exercise_C/index.css:5]: Great work using flex.

Another way that you could complete this without using using margin.
Increase the height of you container, on the red class you can apply align-self: flex-end or something like that.
OR
Change the html and have two containers.

Consider making a box class
and using id's

File: Exercise_D and E/assets/style/index.css [hunk 5]
```diff
@@ -2,3 +2,82 @@
...
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+}
+
+.header {
+    min-height: 100vh;
+    width: 100%;
+    background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(../images/flowershop.jpg);
```
Comment (samoshaughnessy) [inline Exercise_D and E/assets/style/index.css:15]: Decent use of linear gradient to make the image daker, well done

File: Exercise_D and E/assets/style/index.css [hunk 6]
```diff
@@ -2,3 +2,82 @@
...
+    background-position: center;
+    background-size: cover;
+    position: relative;
+}
+
+nav {
+    background-color: white;
+}
+
+.text-box {
```
Comment (samoshaughnessy) [inline Exercise_D and E/assets/style/index.css:25]: If you target the container that encapsulates this class and apply flex, then you can easily put it into the middle of any page

File: Exercise_D and E/assets/style/index.css [hunk 7]
```diff
@@ -2,3 +2,82 @@
...
+    left: 50%;
+    transform: translate(-50%, -50%);
+    text-align: center;
+}
+
+.text-box h1 {
+    font-size: 70px;
+    padding-bottom: 30px;
+}
+
```
Comment (samoshaughnessy) [inline Exercise_D and E/assets/style/index.css:39]: Try to stick to one colour convention throughout your whole css stylesheets

File: Exercise_D and E/assets/style/index.css [hunk 8]
```diff
@@ -2,3 +2,82 @@
...
+    position: relative;
+    cursor: pointer;
+    text-decoration: none;
+    color: white;
+}
+
+.learnMoreBtn:hover {
+    border: 1px solid red;
+    background: red;
+    transition: 1s;
```
Comment (samoshaughnessy) [inline Exercise_D and E/assets/style/index.css:54]: Nice use of transition here! Well done

File: Exercise_D and E/assets/style/index.css [hunk 9]
```diff
@@ -2,3 +2,82 @@
...
+    width: 100%;
+    margin: auto;
+    text-align: center;
+    padding-top: 100px;
+    background: lightgreen;
+    background-size: cover;
+    position: relative;
+    display: grid;
+    grid-template-columns: auto auto;
+}
```
Comment (samoshaughnessy) [inline Exercise_D and E/assets/style/index.css:67]: Nice use of the CSS grid, you have done a vey decent job!

File: Exercise_D and E/index.html [hunk 10]
```diff
@@ -1,22 +1,51 @@
...
+    <nav>
+      <a href="index.html"><img src="./assets/images/logo.png"></a>
+    </nav>
+    <div class="text-box">
+      <h1>Beautiful Flowers</h1>
+      <a href="http://www.9gag.com" target="_blank" class="learnMoreBtn">Learn More</a>
+    </div>
+
+  </section>
+
```
Comment (samoshaughnessy) [inline Exercise_D and E/index.html:28]: Great use of nested elements here! well done

File: practice.js [hunk 11]
```diff
@@ -1,3 +1,8 @@
 // Your Javascript practice exercise goes here
 
 // Please add your if the statements are true or false here!
+x == y //=> false
+x && y < 10 //=> true
+x || y <= 8 //=> true
+y === '3'//=> false
+!(x == 5) //=>true
```
Comment (samoshaughnessy) [inline practice.js:8]: Great job

**Pull request closed without merging**